packages/tasit-action/src/ContractEventsSubscription.js

```diff
@@ -1,15 +1,29 @@
 import Subscription from "./Subscription";
 
 class ContractEventsSubscription extends Subscription {
+  #ethersContract;
   #contract;
 
   // Note: We're considering listening multiple events at once
   //    adding eventName, listener params to constructor.
-  constructor(contract) {
-    super(contract);
+  constructor(ethersContract, contract) {
+    super(ethersContract);
+    this.#ethersContract = ethersContract;
     this.#contract = contract;
   }
 
+  // TODO: Make protected
+  _emitErrorEvent(error, eventName) {
```

Comment:
I think we can remove `eventName` from this one

Reply:
But since this is `Outdated` now, maybe it doesn't matter

Reply:
If you agree it's Outdated and no longer relevant, feel free to resolve it

Reply:
Resolving.